### DISCUSSION TITLE
How to build a 3D model with arbitrary shape？

### DISCUSSION THREAD

**Discussion by crest621010** [neperfepx/neper#688] [Q&A]
Dear all,
How could I build a 3D model with the shape like as follow:
![image](https://user-images.githubusercontent.com/126832043/233818522-acf60263-f1f6-4c33-9184-3c4cefe0717c.png)

**Comment by rquey** [comment]
It is not possible.

You can use `-transform "cut(...)"` to cut the domain by geometrical primitives, but it won't allow you to get this shape.
